README-K8S.md

```diff
@@ -0,0 +1,173 @@
+located in ```/etc/cni/net.d``` . If there are multiple CNI configuration files in the directory, the first one in 
+lexicographic order of file name is used. So make sure to name the netconf file with proper order. 
+
+For example (/etc/cni/net.d/01-infoblox-ipam.conf):
+
+```
+{
+    "name": "infoblox-ipam",
+    "type": "macvlan",
+    "master": "eth0",
+    "ipam": {
+        "type": "infoblox",
+        "subnet": "10.0.0.0/24",
+
+                "routes": [
+                        {"dst": "172.18.0.0/24",
+                         "gw":"10.0.0.1"}
+                ],
+                "network-view": "cni_view"
+    }
+}
+```
+Note : The following type of networks are supported out of the box:
+ ```
+       host-local
+       bridge
+       ipvlan
+       macvlan
+```  
+To run macvlan, the promiscuous mode in eth0 should be enabled on each node. The following command
+can be used to achieve it : ``ip link set eth0 promisc on``
+
+Infoblox IPAM Driver Configuration
+----------------------------------
+The Infoblox IPAM Driver is comprised of two components:
+- Infoblox IPAM Plugin (infoblox):
+  This is the plugin executable specified as the IPAM type in the netconf. This is executed by CNI as a network
+plugin and, by default in a kubernetes environment, is located in the ```/opt/cni/bin``` directory.
+- Infoblox IPAM Daemon (infoblox-cni-daemon):
+  This is the component that interfaces with Infoblox to perform the IPAM functions. This is typically deployed
+as a kubernetes daemonset on each node.
+
+Running the IPAM Daemon
+-----------------------
+The IPAM Daemon accepts the following command line arguments, which specifies Infoblox Grid settings, IPAM Driver
+settings and IPAM Policy settings respectively. Each one of the IPAM Policy settings is the fallback that take
+effect when the same setting have not been specified in the network configuration file.
+
+```
+## Infoblox Grid Settings ##
+--grid-host string
+	IP of Infoblox Grid Host (default "192.168.124.200")
+--wapi-port string
+	Infoblox WAPI Port (default "443")
+--wapi-username string
+	Infoblox WAPI Username (default "")
+--wapi-password string
+	Infoblox WAPI Password (default "")
+--wapi-version string
+	Infoblox WAPI Version (default "2.0")
+--ssl-verify string
+	Specifies whether (true/false) to verify server certificate. If a file path is specified, it is assumed to be a certificate file and will be used to verify server certificate. (default "false")
+
+## IPAM Driver Settings ##
+--socket-dir string
+	Directory in which Infobox IPAM daemon socket is created (default "/run/cni")
+--driver-name string
+	Name of the IPAM driver. This is the file name used to create Infoblox IPAM daemon socket, and has to match the name specified as IPAM type in the CNI configuration. (default "infoblox")
+
+## IPAM Policy Settings ##
+--network-view string
+	Infoblox Network View (default "default")
+--network-container string
```

Comment:
Can you change this document accordingly as we discussed.

Reply:
The network-container argument has description about comma separated subnets with the new commit